projects.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Projects using sButtons - sButtons</title>
+    <link href="assets/css/main.css" rel="stylesheet" />
+    <link href="assets/css/header.css" rel="stylesheet" />
+    <link href="assets/css/footer.css" rel="stylesheet" />
+    <link href="assets/css/header.css" rel="stylesheet" />
+    <link href="assets/css/variables.css" rel="stylesheet" />
+    <link rel="stylesheet" href="assets/css/projects.css" />
+    <!-- bootstrap and fontawesome css -->
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+    <!-- favicon -->
+    <link rel="icon" type="image/png" href="favicon.png" />
+
+    <!-- seo friendly og tags-->
+    <meta
+      name="description"
+      content="Simple buttons you can use easily for your next project."
+    />
+    <meta
+      console.log(projects);
+      name="keywords"
+      content="css, less, buttons, simple, project, github, open source, contribute, first timers, animation, hover, basic, social, login"
+    />
+    <meta name="og:url" content="https://sButtons.github.io/sbuttons/" />
+    <meta
+      name="og:description"
+      content="Simple buttons you can use easily for your next project."
+    />
+    <meta name="og:image" content="site logo url" />
+    <meta name="og:type" content="website" />
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=UA-149400351-4"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "UA-149400351-4");
+    </script>
+  </head>
+  <body>
+    <header>
+      <div class="navbar fixed-top">
+        <div class="navbar-brand">
+          <a href="index.html">
+            <img
+              src="assets/img/sbuttons-icon.png"
+              class="img-fluid"
+              width="64px"
+            />
+          </a>
+        </div>
+        <!-- dropdown menu on mobile -->
+        <div class="btn-group dropstart">
```

Comment:
Nest this under `.navbar-brand`

Reply:
Ok, @shahednasser. Sorry about the late response